llif/exec.go

```diff
@@ -0,0 +1,34 @@
+package llif
+
+import (
+	"github.com/nabla-containers/runnc/libcontainer/configs"
+)
+
+type ExecGenericInput struct {
+	// ContainerId is the id of the container
+	ContainerId string
```

Comment:
Is it worth making ContainerId and ContainerRoot their own struct that gets added to these structs? It's used in 3 different files that I looked at. I'm not sure if it is worth it at all, just bringing it up as a possibility

Reply:
Yea - I agree. This is something that we have to look at. Right now, I've put the entire kitchen sink into every module, but we have to figure out how to filter down to the bare essentials to make the interfaces much cleaner. (Also, possibly removing certain steps as well).